cypress/support/bot.ts

```diff
@@ -18,28 +18,56 @@ limitations under the License.
 
 import request from "browser-request";
 
-import type { MatrixClient } from "matrix-js-sdk/src/client";
+import type { MatrixClient, Room } from "matrix-js-sdk/src/matrix";
 import { SynapseInstance } from "../plugins/synapsedocker";
 import Chainable = Cypress.Chainable;
 
+interface ICreateBotOpts {
```

Comment:
IIRC, the consensus from code style discussions was that we're not starting interface names with `I` anymore.